qiskit/providers/ibmq/credentials/configrc.py

```diff
@@ -122,20 +148,23 @@ def _section_name(credentials_: Credentials) -> str:
 
 def store_credentials(
         credentials: Credentials,
+        default_provider: Optional[str] = None,
```

Comment:
This should take a `HubGroupProject` instead of a `str`, as it should be this module/method's responsibility to know what the stored format is, not the caller's.